### DISCUSSION TITLE
How to register

### DISCUSSION THREAD

**Discussion by AaronTrazona** [ImageMagick/ImageMagick#2622] [Help]
Forum: https://legacy.imagemagick.org/discourse-server/

I tried to register but it will redirect me in github community.

Thanks

**Comment by snibgo** [comment]
That forum is closed. This forum is the replacement.

**Reply by AaronTrazona** [reply]
Thanks  for the quick reply.

**Comment by fmw42** [comment]
The link works fine for me. but you can only view not post. New posts must be done on Github
